lib/rspec-puppet/support.rb

```diff
@@ -87,21 +87,18 @@ def load_catalogue(type, exported = false, manifest_opts = {})
         hiera_config_value = self.respond_to?(:hiera_config) ? hiera_config : nil
         hiera_data_value = self.respond_to?(:hiera_data) ? hiera_data : nil
 
+        rspec_config_values = [
+            :trusted_server_facts,
+            :disable_module_hiera,
+            :use_fixture_spec_hiera,
+            :fixture_hiera_configs,
+            :fallback_to_default_hiera,
+        ].map { |setting| RSpec.configuration.send(setting) }
+
         build_facts = facts_hash(node_name)
-        catalogue = build_catalog(
-          nodename: node_name,
-          facts_val: build_facts,
-          trusted_facts_val: trusted_facts_hash(node_name),
-          hiera_config_val: hiera_config_value,
-          code: build_code(type, manifest_opts),
-          exported: exported,
-          node_params: node_params_hash,
-          trusted_external_data: trusted_external_data_hash,
-          ignored_cache_params: {
-            hiera_data_value: hiera_data_value,
-            trusted_server_facts: RSpec.configuration.trusted_server_facts,
-          },
-        )
+(??)        catalogue = build_catalog(node_name, build_facts, trusted_facts_hash(node_name), hiera_config_value,
+(??)                                  build_code(type, manifest_opts), exported, node_params_hash, hiera_data_value,
+(??)                                  RSpec.configuration.trusted_server_facts)
```

Comment:
Did this section still need some work? It doesn't look like `rspec_config_values` gets used anywhere anymore.

Reply:
fixed the underlying rebase